src/ml_gym/frontend/dashboard/src/app/datatypes.ts

```diff
@@ -0,0 +1,135 @@
+// =================== API TYPES ======================
+
+interface BaseMessageType {
+    event_id: number;
+}
+
+interface EventInfoType {
+    event_type: string;
+    creation_ts: number;
+}
+
+
+// Job status
+
+export type JobStatusPayloadType = {
+    job_id: number;
+    job_type: string; //<CALC, TERMINATE>
+    status: string; //<INIT, RUNNING, DONE>,
+    experiment_id: string; // <path_to_model>
```

Comment:
Codacy has a fix for the issue: [Replace `····` with `··`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
  experiment_id: string; // <path_to_model>
```